src/components/molecules/MenuList/MenuList.tsx

```diff
@@ -0,0 +1,41 @@
+import { forwardRef } from "react";
+import { ListItem } from "@atoms/ListItem";
+import { ListItemDropDown } from "../ListItemDropDown";
+import { MENU_NAVIGATION } from "./const";
+import { MenuListProps } from "./type";
+
+export const MenuList = forwardRef<HTMLDivElement, Pick<MenuListProps, "navigationType">>(
+  ({ navigationType }, ref) => {
+    return (
+      <div
+        className="flex flex-col gap-y-4"
+        ref={ref}
+      >
+        {MENU_NAVIGATION.filter((nav) => nav.groupId === navigationType).map((nav) =>
+          nav.items.map((item) => {
```

Comment:
For better readability, I usually prefer to filter before the return statement:
```suggestion
  ({ navigationType }, ref) => {
  
  const [menuNavigationItems] = MENU_NAVIGATION.filter((nav) => nav.groupId === navigationType)
  
    return (
      <div
        className="flex flex-col gap-y-4"
        ref={ref}
      >
        {menuNavigationItems.items.map((item) =>
```
I think when declaring and destructuring the items beforehand, the code becomes more readable. What do you think?

Reply:
@macfrei 
Looks Nice!

I'll fix this way.